plenum/test/pool_transactions/test_z_node_key_changed.py

```diff
@@ -28,35 +28,39 @@ def testNodeKeysChanged(looper, txnPoolNodeSet, tdir,
                         tconf, sdk_node_theta_added,
                         sdk_pool_handle,
                         allPluginsPath=None):
+    # 1. Add new node
+    orig_view_no = txnPoolNodeSet[0].viewNo
     new_steward_wallet, new_node = sdk_node_theta_added
 
+    # 2. Stop new node and rotate its keys
     new_node.stop()
     looper.removeProdable(name=new_node.name)
     nodeHa, nodeCHa = HA(*new_node.nodestack.ha), HA(*new_node.clientstack.ha)
     sigseed = randomString(32).encode()
     verkey = base58.b58encode(SimpleSigner(seed=sigseed).naclSigner.verraw).decode("utf-8")
     sdk_change_node_keys(looper, new_node, new_steward_wallet, sdk_pool_handle, verkey)
 
+    # 3. Start the new node back with the new keys
+    logger.debug("{} starting with HAs {} {}".format(new_node, nodeHa, nodeCHa))
```

Comment:
When adding logs to tests I'd prefer using `info` level - it just makes it much more easier to work with filtering